### DISCUSSION TITLE
How to export system logs to collector service?

### DISCUSSION THREAD

**Discussion by shravani154** [open-telemetry/opentelemetry-java-instrumentation#5429] [Q&A]
Hello, I have an application which uses kafka and couchbase. I have been adding the applications logs as events to the span and exporting them to the collector. Is there any way to export  kafka, couchbase, system logs to collector service?

**Comment by trask** [comment]
hi @shravani154, the javaagent will instrument log4j, logback and java.util.logging and send those logs to the collector if you configure `OTEL_LOGS_EXPORTER`

https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#exporters
